content/news/playtest-20150102.md

```diff
@@ -0,0 +1,34 @@
+We wish everyone a happy new year and prepared a new playtest which builds upon our [Halloween release](/news/release-20141029/). The main focus was to improve stability and performance as well as implement unique features of Dune 2000 that distinguish it from the other mods.
```

Comment:
I suggest adding an explicit line break between sentences, so that the `The` isn't isolated at the end of the first line.